script/main.js

```diff
@@ -16,3 +68,111 @@ menuItems.forEach((item) => {
+      modalFeaturedImage.src = work.featuredImage;
+
+      const modalCloseIcon = addElement('a', modalWorkCard, 'close-icon');
+      modalCloseIcon.classList.add('fa');
+      modalCloseIcon.classList.add('fa-times');
+
+      if (isDisplayOrMobile) {
+        const modalWorkCardDisplay = addElement('div', modalWorkCard, 'work-card-display');
+        const modalWorkCardTitle = addElement('h3', modalWorkCardDisplay, 'work-card-title-display');
+        modalWorkCardTitle.innerHTML = work.name;
+
+        const modalWorkCardButton1 = addElement('a', modalWorkCardDisplay, 'modal-work-card-button-display');
+        modalWorkCardButton1.innerHTML = 'See Live';
+        modalWorkCardButton1.href = work.linkToLiveVersion;
+
+        const modalWorkCardButtonImg1 = addElement('img', modalWorkCardButton1, 'modal-work-card-button-img');
+        modalWorkCardButtonImg1.src = './img/see live icon.png';
+
+        const modalWorkCardButton2 = addElement('a', modalWorkCardDisplay, 'modal-work-card-button-display');
+        modalWorkCardButton2.innerHTML = 'See Source';
+        modalWorkCardButton2.href = work.linkToSource;
+
+        const modalWorkCardButtonImg2 = addElement('img', modalWorkCardButton2, 'modal-work-card-button-img');
+        modalWorkCardButtonImg2.src = './img/see source icon.png';
+      } else {
+        const modalWorkCardTitle = addElement('h3', modalWorkCard, 'work-card-title');
+        modalWorkCardTitle.innerHTML = work.name;
+      }
+
+      const modalWorkCardTags = addElement('ul', modalWorkCard, 'work-card-tags');
+      modalWorkCardTags.classList.add('nobull');
+
+      workTechnologies.forEach((technology) => {
+        const modalWorkCardTag = addElement('li', modalWorkCardTags, 'work-card-tag');
+        modalWorkCardTag.innerHTML = technology;
+      });
+
+      const modalWorkCardDescr = addElement('p', modalWorkCard, 'work-card-descr');
+      modalWorkCardDescr.innerHTML = work.description;
+
+      if (!isDisplayOrMobile) {
+        const modalWorkCardButton1 = addElement('a', modalWorkCard, 'modal-work-card-button');
+        modalWorkCardButton1.innerHTML = 'See Live';
+        modalWorkCardButton1.href = work.linkToLiveVersion;
+
+        const modalWorkCardButtonImg1 = addElement('img', modalWorkCardButton1, 'modal-work-card-button-img');
+        modalWorkCardButtonImg1.src = './img/see live icon.png';
+
+        const modalWorkCardButton2 = addElement('a', modalWorkCard, 'modal-work-card-button');
+        modalWorkCardButton2.innerHTML = 'See Source';
+        modalWorkCardButton2.href = work.linkToSource;
+
+        const modalWorkCardButtonImg2 = addElement('img', modalWorkCardButton2, 'modal-work-card-button-img');
+        modalWorkCardButtonImg2.src = './img/see source icon.png';
+      }
+
+      modalCloseIcon.addEventListener('click', () => {
+        modalWorkContainer.classList.add('hide');
+        modalWorkContainer.removeChild(modalWorkContainer.firstElementChild);
```

Comment:
### Your Close Button
![close-button-student](https://user-images.githubusercontent.com/93039359/196651445-d285b761-104c-44bd-90c6-e82cca9092ef.PNG)

### Figma Mobile Button
![Figma-Mobile-Design](https://user-images.githubusercontent.com/93039359/196651495-e376545b-dfec-48b1-935b-92d71ac3637e.PNG)

- [ ] Your close button is not very similar to the Figma template in mobile version. Kindly make it look similar to the Figma template to make your design professional 😊
- [ ] Kindly make your close button at proper position as you can see in the Figma template to make your design catchy 👍